scripts/changelog/templates/challenge_level.md.tera

```diff
@@ -0,0 +1,37 @@
+{%- import "change.md.tera" as m_c -%}
+
+{# This macro convert a merge challenge level into readable output #}
+{%- macro challenge_level(e, changes) -%}
+
+{%- if e >= 5 -%}
+    {%- set level = "‼️ Breaking Changes" -%}
+    {%- set text = "This release contains **breaking changes**. Be sure to upgrade the affected interfaces." -%}
+{%- elif e >= 3 -%}
+    {%- set level = "❗️ Attention" -%}
+    {%- set text = "This release contains some non-trivial updates. Be mindful when upgrading." -%}
+{%- else -%}
+    {%- set level = "Trivial" -%}
+    {%- set text = "This release contains relatively small updates." -%}
+{%- endif %}
+
+
+<!-- detected max level is: {{e}} -->
+
+{%- if level %}
+{{level}}: {{text}}
+
+{% if e >= 3 %}
+The changes motivating this priority level are:
```

Comment:
I think we should rename `priority level` to `difficulty level`

Reply:
Ops, thanks for picking this up

Reply:
So not `challenge` but `difficulty`?

Reply:
Updated :)